### PR TITLE
Read Blacklisted skills from config

### PR DESCRIPTION
**Pull request by JarbasAI**
Blacklisted skills were hard-coded in skills core.py, now read from config

### CODE REVIEW

**Comment by forslund** [comment]
Looks good! I notice a small issue (not yours, it was there before): the name of the `media` skill is actually `skill-media`. Could you update the config to fix this as well?

**Comment by JarbasAI** [comment]
done

File: mycroft/skills/core.py [hunk 1]
```diff
@@ -36,7 +36,9 @@
 from mycroft.skills.settings import SkillSettings
 __author__ = 'seanfitz'
 
-BLACKLISTED_SKILLS = ["send_sms", "media"]
+skills_config = ConfigurationManager.instance().get("skills")
+BLACKLISTED_SKILLS = skills_config.get("blacklisted_skills",[])
```
Comment (forslund) [inline mycroft/skills/core.py]: there's a pep8 issue here. There should be a space between `,` and `[]`.

**Comment by forslund** [comment]
Thank you. Merging!

**Pull request merged**